### PR TITLE
test: increase x/oracle unit test coverage

### PR DESCRIPTION
**Pull request by RafilxTenfen**
## Description

- Add unit test cover to `x/oracle/types/denom.go` and `x/oracle/types/keys.go`

helps: #661

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added appropriate labels to the PR
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: x/oracle/types/keys_test.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umee-network/umee/v2/app"
+	"github.com/umee-network/umee/v2/x/oracle/types"
+)
+
+const (
+	denomLunaIBC = "ibc/0EF15DF2F02480ADE0BB6E85D9EBB5DAEA2836D3860E9F97F9AADE4F57A31AA0"
+	denomAtomIBC = "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
```
Comment (adamewozniak) [inline x/oracle/types/keys_test.go:14]: Rename suggestion : 

```suggestion
	ibcDenomLuna = "ibc/0EF15DF2F02480ADE0BB6E85D9EBB5DAEA2836D3860E9F97F9AADE4F57A31AA0"
	ibcDenomAtom = "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
```

+ can we move these to a central location and use them in both files? maybe to the top of `test_utils.go`

File: x/oracle/types/denom_test.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+var (
+	umeeDenom = types.Denom{
+		BaseDenom:   app.BondDenom,
+		SymbolDenom: app.DisplayDenom,
+		Exponent:    6,
+	}
+	lunaDenom = types.Denom{
+		BaseDenom:   denomLunaIBC,
+		SymbolDenom: "ULUNA",
```
Comment (adamewozniak) [inline x/oracle/types/denom_test.go:19]:
```suggestion
		SymbolDenom: "LUNA",
```

File: x/oracle/types/denom_test.go [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+		Exponent:    6,
+	}
+	atomDenom = types.Denom{
+		BaseDenom:   denomAtomIBC,
+		SymbolDenom: "UATOM",
```
Comment (adamewozniak) [inline x/oracle/types/denom_test.go:24]:
```suggestion
		SymbolDenom: "ATOM",
```

**Review by adamewozniak — COMMENTED** [review]
🚀

File: x/oracle/types/keys_test.go [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+func TestGetAggregateExchangeRatePrevoteKey(t *testing.T) {
+	testCases := []struct {
+		val         sdk.ValAddress
+		expectedKey []byte
+	}{
+		{
+			val:         []byte("addr________________"),
+			expectedKey: []byte{0x4, 0x14, 0x61, 0x64, 0x64, 0x72, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f, 0x5f},
```
Comment (toteki) [inline x/oracle/types/keys_test.go:83]: Could add a comment as to the origin of these bytes - that would also clarify what the meaning would be if the test ever failed.

Comment (toteki) [reply x/oracle/types/keys_test.go:83]: For example:
```
// keyPrefixAggregateExchangeRatePrevote | lengthPrefixed(addr)
```

**Comment by adamewozniak** [comment]
Changelog entry pls! Don't [be like me](https://github.com/umee-network/umee/pull/784)

**Review by toteki — APPROVED** [review]

**Review by adamewozniak — APPROVED** [review]

**Pull request merged**